calico-enterprise/multicluster/federation/kubeconfig.mdx

```diff
@@ -132,34 +73,43 @@ Create a kubeconfig file, for each cluster, that will be used by other clusters
        kubernetes.io/service-account.name: "tigera-federation-remote-cluster"
    EOF
    ```
-   - Retrieve the ServiceAccount token value and replace `<YOUR-SERVICE-ACCOUNT-TOKEN>` with it's value:
-   ```bash
-   kubectl describe secret tigera-federation-remote-cluster -n calico-system
-   ```
-
-   #### If using Kubernetes < 1.24
-   - Retrieve the ServiceAccount token value and replace `<YOUR-SERVICE-ACCOUNT-TOKEN>` with it's value:
-   ```bash
-   kubectl describe secret -n calico-system $(kubectl get serviceaccounts tigera-federation-remote-cluster -n calico-system -o jsonpath='{.secrets[0].name}')
-   ```
 
-1. Retrieve and save the certificate authority and server data:
-
-   Run the following command:
-   ```bash
-   kubectl config view --flatten --minify
+1. Use the following command to create your kubeconfig at `$KUBECONFIG_NAME`:
+   ```yaml
+    echo "apiVersion: v1
+    kind: Config
+    users:
+      - name: tigera-federation-remote-cluster
+        user:
+          token: $(kubectl get secret tigera-federation-remote-cluster -n kube-system -o=jsonpath='{.data.token}' | base64 -d)
+    clusters:
+      - name: tigera-federation-remote-cluster
+        cluster:
+          certificate-authority-data: $(kubectl config view --raw -o=jsonpath='{.clusters[0].cluster.certificate-authority-data}')
+          server: $(kubectl config view --raw -o=jsonpath='{.clusters[0].cluster.server}')
+    contexts:
+      - name: tigera-federation-remote-cluster-ctx
+        context:
+          cluster: tigera-federation-remote-cluster
+          user: tigera-federation-remote-cluster
+    current-context: tigera-federation-remote-cluster-ctx" > $KUBECONFIG_NAME
    ```
-   Replace `<YOUR-CERTIFICATE-AUTHORITY-DATA>` and `<YOUR-SERVER-ADDRESS>` with `certificate-authority-data` and `server` values respectively.
 
 1. Verify that the `kubeconfig` file works:
 
-   Issue a command like the following to validate the kubeconfig file can be used to connect to the current cluster and access resources:
+   Issue the following command to validate the kubeconfig file can be used to connect to the current cluster and access resources:
    ```bash
-   kubectl --kubeconfig=kubeconfig-app-a get nodes
+   kubectl --kubeconfig=$KUBECONFIG_NAME get nodes
    ```
 
+Once you've created a kubeconfig for **each** cluster, you can proceed to the next section to establish the cluster connections that form the mesh.
+
 ### Create RemoteClusterConfigurations
```

Comment:
Out of scope for this PR. But I find myself wondering about the two tabs below. We offer no explanation for the two types. How do I know which set of instructions to follow?

Reply:
Also, the idea of "for each pair" is not clear to me. If I'm doing this with 12 clusters, then am I looping through this procedure with every possible pair?